elizabeth-1/README.md

```diff
@@ -0,0 +1,225 @@
+# Tgrade: elizabeth-1 Community Testnet
+
+Elizabeth-1 is our community managed testnet.  It is an enviroment for the community to build and test their Trusted Circle and applications on.
+
+## Hardware Requirements
+For running a tgrade validator. We tested successfully with the following Architecture:
+
+- Ubuntu 20.04 LTS
+- go version 1.19.1 [1] or newer
+- Installed packages make and build-essential [2] [3]
+- 2 or more CPU cores Intel or AMD chipset
+- At least 40GB of disk storage
+- At least 4GB of memory (RAM)
+
+## Prerequists
+
+Update Go to a recent version 
+
+```
+sudo rm -rf /usr/local/go && cd ~/ && \
+sudo wget https://go.dev/dl/go1.19.1.linux-amd64.tar.gz && \
+sudo tar -C /usr/local -xvf go1.19.1.linux-amd64.tar.gz && \
+sudo chown -R $USER:$USER /usr/local/go && rm -f go1.19.1.linux-amd64.tar.gz
+```
+Add the following to the end of your `.bashrc` and run
+
+```
+nano ~/.bashrc
+```
+```
+export GOROOT=/usr/local/go
+export GOPATH=$HOME/go
+export GO111MODULE=on
+export PATH=$PATH:/usr/local/go/bin:$HOME/go/bin
+```
+```
+source .bashrc
+```
+
+## Build the tgrade binary
+The tgrade binary is the backbone of the platform. 
+```
+cd ~
+git clone https://github.com/confio/tgrade
+cd tgrade
+git checkout v2.0.2
+make build
+```
+
+Move the binary to an executable path
+```
+sudo mv build/tgrade /usr/bin
+```
+Check that you have the right `tgrade` version installed
+```
+tgrade version --long
+```
+This should return:
+```
+name: tgrade
+server_name: tgrade
+version: 2.0.2
+commit: a990225d4b92f09d7b6fad3d6147d71c3711ece4
+build_tags: netgo,ledger
+go: go version go1.19.1 linux/amd64
+```
+
+## Setting up a Testnet Tgrade Validator
+
+### Initialize your genesis and configuration files
+Initialize your genesis and configuration files for your validator node
+
+```
+tgrade tendermint unsafe-reset-all --home $HOME/.tgrade
+tgrade init "<NAME_OF_MY_VALIDATOR>" --chain-id elizabeth-1
+```
+
+#### Get the genesis file ####
+```
+cd $HOME/.tgrade/config
+rm genesis.json
+wget https://github.com/lightiv/tgrade/raw/main/Elizabeth-1/genesis.json -O ~/.tgrade/config/genesis.json
```

Comment:
This would need to be updated I guess.
```suggestion
wget https://github.com/confio/tgrade-networks/raw/main/elizabeth-1/genesis.json -O ~/.tgrade/config/genesis.json
```